packages/cli-app/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../tsconfig/base.json",
```

Comment:
This file doesn't exist in this repo. Here are the original contents:

```js
{
  "compilerOptions": {
    "target": "es2020",
    "lib": ["dom", "dom.iterable", "esnext"],
    "noImplicitAny": false,
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": false,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve",
    "incremental": true,
    "baseUrl": "./src"
  },
  "exclude": ["node_modules"]
}
```

Reply:
Thanks! I haven't really put work into cli-app in this branch, I was thinking of importing (and rename it if necessary) it in a first PR to keep the commit history (https://github.com/grafbase/grafbase/pull/820).